src/main/java/dinkplugin/notifiers/GrandExchangeNotifier.java

```diff
@@ -106,8 +120,21 @@ private void handleNotify(int slot, GrandExchangeOffer offer) {
     }
 
     private boolean shouldNotify(int slot, GrandExchangeOffer offer) {
-        if (initTicks.get() > 0)
-            return false;
+        // On login, ignore updates if they aren't completions that haven't been observed before
```

Comment:
We only care about offers that have been completed and *have not* been observed by the RuneLite GE plugin.

We only want to fire the notification once, so we only fire if the completion hasn't been observed before. If it has been observed, we ignore the event because we could've already sent a notification for it.